### PR TITLE
Doc: warn of using client tokens

### PR DESCRIPTION
**Pull request by cloudybyte**

### CODE REVIEW

File: src/lib/config.rs [hunk 1]
```diff
@@ -7,9 +7,13 @@ use serenity::model::id;
...
 pub struct Config {
+    #[serde(rename = "DISCORD_TOKEN")]
     pub discord_token: String,
+    #[serde(rename = "SPOTIFY_USERNAME")]
     pub spotify_username: String,
+    #[serde(rename = "SPOTIFY_PASSWORD")]
     pub spotify_password: String,
+    #[serde(rename = "DISCORD_USER_ID")]
     pub discord_user_id: id::UserId,
 }
```
Comment (codetheweb) [inline src/lib/config.rs:18]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -17,7 +17,10 @@ Aoede is a Discord music bot that **directly** streams from **Spotify to Discord
 
 (Images are available for x86 and arm64.)
 
-**Docker Compose (recommended)**:
+### Note:
+⚠️ Aoede only officially supports bot tokens. Providing a user token probably won't work
```
Comment (codetheweb) [inline README.md:21]:
```suggestion
⚠️ Aoede only supports bot tokens. Providing a user token won't work.
```

**Review by codetheweb — CHANGES_REQUESTED** [review]
Thank you!

File: src/lib/config.rs (hunk 1 above)
Comment (cloudybyte) [reply src/lib/config.rs:18]: oh that slipped through when switching branches, that shouldn't be merged and was already handled in the other PR

**Review by codetheweb — APPROVED** [review]

**Pull request merged**